tests/module_test.py

```diff
@@ -1,6 +1,3 @@
-from pyteal import *
```

Comment:
This should not be removed. The below test is to make sure this line does not shadow the `int` variable

Reply:
👍